src/olympia/accounts/views.py

```diff
@@ -271,3 +274,58 @@ def retrieve(self, request, *args, **kwargs):
             # address has an account associated with it.
             user = STUB_FXA_USER
         return Response(self.get_serializer(user).data)
+
+
+class AccountSuperCreate(JWTProtectedView):
+
+    @waffle_switch('super-create-accounts')
+    def post(self, request):
+        if not acl.action_allowed_user(request.user,
```

Comment:
Looks like `olympia.api.permissions.GroupPermission` would do it.

https://github.com/mozilla/addons-server/blob/ce3c249102154bea933a7f63975c279f82db76f1/src/olympia/api/views_drf.py#L417